tests/testsuite.src/run_misc.at

```diff
@@ -15226,3 +15226,26 @@ AT_CHECK([$COMPILE prog.cob])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([DISPLAY FLOAT no leading zeroes in exponent])
```

Comment:
just go with "DISPLAY FLOAT" here - and please add keywords FLOAT-SHORT FLOAT-LONG COMP-1 COMP-2 as AT_KEYWORDS